deezer/resources.py

```diff
@@ -55,6 +55,26 @@ def get_relation(self, relation, **kwargs):
         return self.client.get_object(self.type, self.id, relation,
                                       self, **kwargs)
 
+    def iter_relation(self, relation, **kwargs):
+        """
+        Generic method to iterate relation from any resource.
+
+        Query the client with the object's known parameters
+        and try to retrieve the provided relation type. This
+        is not meant to be used directly by a client, it's more
+        a helper method for the child objects.
```

Comment:
To reflect the docstring into the code could you change the name of the method to start by '_' as per the python conventions.